index.js

```diff
@@ -1,10 +1,11 @@
 const path = require('path');
 const core = require('@actions/core');
 const { CodeDeploy, waitUntilDeploymentSuccessful } = require('@aws-sdk/client-codedeploy');
-const { ECS, waitUntilServicesStable } = require('@aws-sdk/client-ecs');
+const { ECS, waitUntilServicesStable, waitUntilTasksStopped } = require('@aws-sdk/client-ecs');
 const yaml = require('yaml');
 const fs = require('fs');
 const crypto = require('crypto');
+const { default: cluster } = require('cluster');
```

Comment:
What is this import doing?

Reply:
Isn't this specific instance referencing the NodeJS [cluster](https://nodejs.org/api/cluster.html)?